Tests/MapboxMapsTests/Foundation/Camera/CameraAnimatorMock.swift

```diff
@@ -0,0 +1,24 @@
+import XCTest
+
+#if canImport(MapboxMaps)
+@testable import MapboxMaps
+#else
+@testable import MapboxMapsFoundation
+#endif
+
+final class CameraAnimatorMock: CameraAnimatorDelegate {
```

Comment:
Shouldn't this be `CameraAnimatorDelegateMock` or `CameraAnimatorDelegateStub` ?

Reply:
yes